travis/travis_weblate.py

```diff
@@ -67,17 +68,31 @@ def _apply_patch_odoo(self):
         """This patch is necessary because the weblate does not check which
         word and the translated are the same to use it in its percentage of
         translated"""
-        paths = [os.path.join('openerp', 'tools', 'translate.py'),
-                 os.path.join('odoo', 'tools', 'translate.py')]
-        for path in paths:
-            s_file = os.path.join(self._server_path, path)
-            if not os.path.isfile(s_file):
-                continue
-            cmd = ["sed", "-i", "-e",
-                   r"s/translation'] = src/translation'] = ''/g",
-                   s_file]
-            print " ".join(cmd)
-            subprocess.call(cmd)
+        for base in ('odoo', 'openerp'):
+            p_file = os.path.join(self._server_path, base,
+                                  os.path.join('tools', 'translate.py'))
+            if os.path.isfile(p_file):
+                sed = ["sed", "-i", "-e",
+                       r"s/translation'] = src/translation'] = ''/g", p_file]
```

Comment:
@moylop260 
Thanks for teach me this way to use multiple folders in `rgrep` command
I try to use this example into the python and i can not made to work

![image](https://user-images.githubusercontent.com/1387970/27931202-688817d6-6267-11e7-8f32-2a8dd6e028dd.png)

I the correct way to run this command ?